app/src/main/java/org/systers/mentorship/view/fragments/EditProfileFragment.kt

```diff
@@ -51,13 +52,13 @@ class EditProfileFragment: DialogFragment() {
             if (successful != null) {
                 if (successful) {
                     Toast.makeText(context,getText(R.string.update_successful),Toast.LENGTH_LONG).show()
+                    profileViewModel.getProfile()
```

Comment:
@m-murad I'm ok with this but just out of curiosity, between fetching the data again or setting the view with the updated data (once the result is successful), which one may be a better option? When I did this feature I avoided fetching all data again.

Reply:
Hey @isabelcosta I know you asked @m-murad but according to me once the result is successful then only it is appropriate to fetch the data again because when the result is successful it means that profile has been successfully updated and that will fetch the updated data.
Let's wait for @m-murad opinion :+1:

Reply:
@mohak1283 if you have an opinion on this feel free to share it! I just mentioned @m-murad because he already did a review, so I wanted to know if he noticed this choice of yours, that as I said I'm ok with it, I just want to learn a reason to prefer fetching data again over updating locally the view.

Reply:
I did think about this. 
We will be adding images later (backend will publish images to S3 bucket). We would need image URL to be saved, that can only happen if we get the data again.

If we didn't have an image then updatig the views with local data would have been fine.

Reply:
@m-murad Ok perfect! thank you for clarifying that for us! Will approve this and merge it ;)

Reply:
> backend will publish images to S3 bucket

So does this mean that the app will upload profile pictures to the backend server instead of the S3 Bucket?
@m-murad

Reply:
yes @haroldadmin I didn't understand this at first, but yes.